FromAmazonCorrettoJdk11.Dockerfile

```diff
@@ -47,4 +56,4 @@ ENV JAVA_OPTS="-Xms1g -Xmx1g"
 #https://github.com/krallin/tini
 # -v -vv and -vvv stands for verbosity level
 ENTRYPOINT ["/sbin/tini", "-v", "--", "/opt/service/entry-point.sh"]
-CMD [--spring.profiles.active=local]
\ No newline at end of file
+CMD [--spring.profiles.active=local]
```

Comment:
what has changed here?

Reply:
i think this is because of the some blank lines above.